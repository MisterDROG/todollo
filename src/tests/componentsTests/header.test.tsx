import Header from "../../components/header/header"
import { render, screen } from '@testing-library/react'

describe('Header tests', () => {
    test('Is header renders right', () => {

        const { container } = render(<Header />)
        const divLogoContainer = container.querySelector('.header__logo-container')
        const pLogoText = screen.getByText(/ToDollo board/i)

        expect(pLogoText).toBeInTheDocument()
        expect(divLogoContainer).toBeInTheDocument()
        expect(container).toMatchSnapshot()
    })
})