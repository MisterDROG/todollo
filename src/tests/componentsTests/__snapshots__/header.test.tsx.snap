// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests Is header renders right 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="header__logo-container"
    >
      <img
        alt="ToDollo Logo"
        class="header__logo-img"
        src="TodolloDarkLogo.png"
      />
      <p
        class="header__logo-text"
      >
        ToDollo board
      </p>
    </div>
  </div>
</div>
`;
